MapboxNavigation/RouteMapViewController.swift

```diff
@@ -409,7 +409,7 @@ extension RouteMapViewController: NavigationMapViewDelegate {
         }
         
         let nearByCoordinates = routeController.routeProgress.currentLegProgress.nearbyCoordinates
-        let closest = closestCoordinate(on: nearByCoordinates, to: location.coordinate)!
+        guard let closest = closestCoordinate(on: nearByCoordinates, to: location.coordinate) else { return location }
```

Comment:
This function would only return `nil` if `nearByCoordinates` is an empty array. Is it legitimate for there to be no `nearbyCoordinates` in the current leg?